libr/anal/var.c

```diff
@@ -694,15 +702,16 @@ R_API void extract_rarg(RAnal *anal, RAnalOp *op, RAnalFunction *fcn, int *reg_s
 				char *vname = r_str_newf ("%s%d", "arg", i + 1);
 				r_anal_var_add (anal, fcn->addr, 1, delta, 'r', NULL, anal->bits / 8,
 						1, vname);
+				if (op->var && op->var->kind != 'r') {
```

Comment:
What do you think to create a enum/constant for op->var->kind values? Will be more self-explanatory, imho.

Reply:
It's already there , i didin't use it , i will update it :)

Reply:
@sivaramaaa you can do it in the next PR then.